pages/specification/json-hyper-schema/_index.md

```diff
@@ -13,6 +13,531 @@ In essence:
 - JSON Hyper-Schema is helpful in complex APIs where clients need to define and explicitly understand the relationships between resources and actions, especially when navigating resources without prior knowledge of the API structure.
 - It helps create more discoverable and self-documenting APIs, making it easier for clients to interact with them.
 
+
+## Hyper-schema use cases
+
+### Basic concept
+
+**Get a JSON instance**
+
+```http
+GET https://example.com/book/12345
+```
+
+_This `GET` request fetches information about a specific book, identified by the `book` ID (`12345`). In response:_
+
+```http
+200 OK
+Content-Type: application/json
+Link: <https://example.com/schemas/book>; rel="describedby"
+
+{
+  "title": "Mobby-Dick",
+  "authorId": 100,
+  ...
+}
+```
+
+_The server responds with a successful `200 OK` status, JSON-formatted data, and a `Link` header that references the schema describing the structure of the returned data, including details like the `book` `title` and a link to the author via `authorId`._
+
+**Get the schema that describes the instance using the "`describedby`" Link header.**
+
+```http
+GET https://example.com/schemas/book
+```
+
+_This request fetches the schema describing the structure of the `book` object.
+In response:_
+
+```http
+200 OK
+Content-Type: application/schema+json
+
+{
+  "links": [
+    {
+      "href": "/author/{authorId}",
+      "rel": "author",
+      "templateRequired": ["authorId"]
+    }
+  ]
+}
+```
+
+_The response contains a JSON Schema describing the instance's structure and links to relevant resources, such as the author's resource via `rel="author`." It also specifies that the `authorId` is required to complete the URL template._
+
+**Use the LDO combined with the instance to construct a link to get information about the author.**
+
+```http
+GET https://example.com/author/100
+```
+
+This request fetches data for the author with `authorId` 100. In response:
+
+```http
+200 OK
+Content-Type: application/json
+Link: <https://example.com/schemas/author>; rel="describedby"
+
+{
+  "name": "Herman Melville",
+  ...
+}
+```
+
+_The response includes a JSON object with the author's details, and the `Link` header points to the schema that describes the author, following the same format as the book schema._
+
+Keep following links to discover more content.
+
+### Actions
+
+#### Search
+
```

Comment:
issue(blocking): Most use cases jump straight into the examples without giving much context. I think the document could benefit from adding more context and linking to other resources readers can use to expand their knowledge about the hyper-schema. This is especially important for readers who are new to the hyper-schema vocabulary.